Server/Test_Plan_Federated_Sharing.md

```diff
@@ -28,53 +28,53 @@ Previous requirements
 | Test Case                                | Expected Result                          | Result         | Related Comment |
 | ---------------------------------------- | ---------------------------------------- | -------------- | --------------- |
 | **Federated share**                      |                                          |                |                 |
-| share a file (server A) to (server B)    | shared file can be seen from the server B | :construction: |                 |
-| share a folder (server A) to (server B)  | shared folder can be seen from the server B and access to its contents | :construction: |                 |
-| unshare a file / folder in your owncloud (server B) | shared file is no longer visible from the server B | :construction: |                 |
-| share a file (server A) in (server B), delete file in server A | shared file is no longer visible from the server B | :construction: |                 |
-| share a file password-protected (server A) with the correct password (server B) | shared file can be seen from the server B | :construction: |                 |
+| share a file (server A) to (server B)    | shared file can be seen from server B    | :construction: |                 |
```

Comment:
If text takes up less than the "defined" column width at the top of the table, then people had been putting that amount of space to make the "vertical bar" columns line up in a text editor. So I did the same. It should make no difference to the final rendering either way.